cluster/config-defaults.yaml

```diff
@@ -1133,7 +1133,11 @@ open_sg_ingress_ranges: ""
 
 # Each DNS label (subdomain) can be 63 octets or less (https://datatracker.ietf.org/doc/html/rfc1035#section-2.3.4)
 # This custom value sets the subdomain max allowed length taking into consideration the 'cname-' prefix added by external-dns
+{{ if eq .Cluster.Alias "stups-test" }}
```

Comment:
Since we are targeting e2es with this change lets rather make this clear with:

`{{if eq .Cluster.Environment "e2e"}}`

Reply:
Nevermind, since the e2es run in `stups-test` and not e2e clusters :facepalm: